Python/Bindings/Processor/ProcessorPybind.hh

```diff
@@ -40,7 +40,7 @@ namespace NymphPybind
     {
         py::class_< Nymph::Processor, PyProcessor, std::shared_ptr<Nymph::Processor> >(nymphProcessor, "_Processor")
                 .def(py::init<const std::string& >())
-                .def("configure", &Nymph::Processor::Configure)
+                .def("Configure", &Nymph::Processor::Configure)
```

Comment:
I think this should be lower-case `configure`, and then in the trampoline class use the macro that's appropriate for situations where the Python and C++ names don't match.